api/src/test/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactoryTest.java

```diff
@@ -205,6 +242,22 @@ private ReportClaimMessageContext aReportClaimMessageContext(LocalDateTime times
                 .build();
     }
 
+    private ReportPaymentMessageContext aReportPaymentMessageContext(LocalDateTime timestamp,
```

Comment:
Test data factory?

Reply:
Wouldn't make sense here as every value is being explicitly set to a value that we're testing against.